en/02_Developer_Guides/08_Performance/08_Other_Optimisations.md

```diff
@@ -0,0 +1,74 @@
+
+> [!NOTE]
+> Most shared hosting providers will have maximum values that can't be altered.
+
+For certain tasks like synchronizing a large `assets/` folder with all file and folder entries in the database, more
+resources are required temporarily. In general, we recommend running resource intensive tasks through the
+[command line](../cli), where configuration defaults for these settings are higher or even unlimited.
+
+```php
+// app/src/Services/MyClass.php
+namespace App\Services;
+
+use SilverStripe\Core\Environment;
+
+class MyClass
+{
+    public function increaseLimits(): string
+    {
+        Environment::increaseMemoryLimitTo('256M');
+        Environment::increaseTimeLimitTo(500);
+    }
+}
+```
+
+## Cache warming
+
+For sites employing caching, cache warming involves proactively populating the cache with frequently accessed data. This reduces cache misses and ensures rapid response speeds from the outset, providing users with up-to-date content efficiently.
+
+Here are some strategies for cache warming:
+
+- Generate cache entries for the most popular pages or data on deployment, for instance by implementing the [`Flushable`](api:SilverStripe\Core\Flushable) on a class that then pre-fetches data.
+- Use a cron job to periodically regenerate cache entries.
+- Warm the cache whenever data is changed or added, such as by implementing the `onAfterWrite()` extension hook on your `DataObject`.
+
+## Enable opcode cache
+
+Enable an [opcode](https://en.wikipedia.org/wiki/Opcode) cache such as [OPcache for PHP]([OPcache](https://www.php.net/manual/en/book.opcache.php)) to increase the performance of PHP code.
+
+When PHP code is executed, it is first compiled into intermediate bytecode (opcodes). Without an opcode cache, this compilation occurs on every request. An opcode cache stores these compiled opcodes in shared memory, eliminating the need for repeated compilation and significantly reducing execution time for subsequent requests.
+
+## Use a CDN
+
+You can use a third-party Content Delivery Network (CDN) to cache and serve static assets (like images, JavaScript, and CSS) from servers geographically closer to your users. This has a few key benefits:
+
+- Static assets(images, JavaScript, CSS) are delivered from an edge server, speeding up load times.
+- Decreases traffic to your main server, improving its performance.
+- Can provides additional security features like DDoS protection
+
+CDN's will typically respect existing [HTTP cache headers](http-cache-headers) which you can configure within your application.
+
+## Use a reverse proxy
+
+Similar to a CDN, you can use a reverse proxy as an intermediary server positioned in front of your application. It intercepts client requests, offering significant performance and security benefits.
```

Comment:
```suggestion
Similar to a CDN, you can use a reverse proxy as an intermediary server positioned in front of your application. It intercepts client requests and can offer significant performance and security benefits if configured correctly.
```
Just making it a little clearer that it's not a magic bullet - if you don't do it right you don't get the benefits